api/Document.json

```diff
@@ -11310,10 +11332,12 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": false
+              "version_added": "1",
+              "version_removed": true
```

Comment:
This seems right per https://bugzilla.mozilla.org/show_bug.cgi?id=687426, but can you find the removal version and add here?

The same goes for the follow document.xml* entries, and they might not have been removed together.